packages/assets-controllers/src/CurrencyRateController.ts

```diff
@@ -196,6 +205,9 @@ export class CurrencyRateController extends BaseControllerV2<
    * @returns The controller state.
    */
   async updateExchangeRate(): Promise<CurrencyRateState | void> {
+    if (!this.#enabled) {
+      return this.state;
+    }
```

Comment:
Given that this is a public method and can be used on its own apart from the polling, it might be confusing if this does nothing. What do we think about logging a message here? Something like

``` suggestion
    if (!this.#enabled) {
      console.info('[CurrencyRateController] Not updating exchange rate since network requests have been disabled');
      return this.state;
    }
```